### PR TITLE
added volume configuration (backend, ibackend, frontend) for faster d…

### PR DESCRIPTION
**Pull request by magp18**
…eployment. Added envs in README for the configuration

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -32,6 +32,9 @@ services:
       - CELERY_BROKER_URL=amqp://rabbitmq
     networks:
       - openimis-net
+    volumes:
+      - "${BE_CONF_PATH}:/openimis-be/openimis.json"
+      - "${DEV_CONF_PATH:-/dev/null}:/openimis-dev"
```
Comment (delcroip) [inline docker-compose.yml:37]: Does it work on windows ?

File: windows/docker-compose.yml [hunk 2]
```diff
@@ -45,6 +46,9 @@ services:
       - SITE_ROOT=api
     networks:
       - openimis-net
+    volumes:
+      - "${BE_CONF_PATH}:/openimis-be/openimis.json"
+      - "${DEV_CONF_PATH:-/dev/null}:/openimis-dev"
     depends_on:
       database:
         condition: service_healthy
```
Comment (delcroip) [inline windows/docker-compose.yml:54]: please uncomment the frontend and add the dev_path (dev_conf_path is missleading)

File: windows/docker-compose.yml [hunk 3]
```diff
@@ -45,6 +46,9 @@ services:
       - SITE_ROOT=api
     networks:
       - openimis-net
+    volumes:
+      - "${BE_CONF_PATH}:/openimis-be/openimis.json"
```
Comment (delcroip) [inline windows/docker-compose.yml:50]: we might need to use the override approach https://docs.docker.com/compose/extends/#example-use-case
normal without the config ENV variable and the override with it.

**Review by delcroip — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (magp18) [reply docker-compose.yml:37]: you are right, according to https://stackoverflow.com/questions/313111/is-there-a-dev-null-on-windows it should be 'nul', for windows

File: docker-compose.override.yml [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+  #     - openimis-net
+    ## WARNING:
+    ## exposing the database port outside the openimis-net network
+    ## may lead to security issue (depending on your network topology)
+    # ports:
+    #  - 1433:1433
+  backend:
+    platform: linux
+    container_name: openimis-backend
+    build: ../openimis-be_py
```
Comment (delcroip) [inline docker-compose.override.yml:22]: why not using github link ?

**Review by delcroip — CHANGES_REQUESTED** [review]

File: docker-compose.override.yml (hunk 4 above)
Comment (magp18) [reply docker-compose.override.yml:22]: It's because the master is not using it so when I cloned it, I got it without a GitHub context

**Comment by magp18** [comment]
In the windows docker-compose file, there are two frontend services, is this intended?

**Review by delcroip — APPROVED** [review]

**Pull request merged**